www/src/js/components/installer.js

```diff
@@ -16,11 +16,12 @@ module.exports = React.createBackboneClass({
     var installActionString = model.get('installActionString');
     var isInstalled = model.get('isInstalled');
     var isInstallable = model.get('isInstallable');
+    var isPriced = model.get('priced');
     var installHTMLClass = model.get('installHTMLClass');
 
     var rootDivClass = "b-installer " + installerClass + " " + installHTMLClass;
 
-    if (isInstallable) {
+    if (isInstallable || isPriced) {
```

Comment:
Well yes, installable means really that is can be installed (downloaded + installed) by snapd, isPriced mean "can be bought", both display a button but have different behaviors. the distinction matters there,